inc/saitypes.h

```diff
@@ -828,6 +828,9 @@ typedef enum _sai_acl_stage_t
     /** Pre-ingress Stage */
     SAI_ACL_STAGE_PRE_INGRESS,
 
+    /** Post Ingress Stage */
```

Comment:
Do we need to tie the ACL stage (or placement in the pipeline) to the match_type? There might be use cases where a post-ingress stage needs to be implemented with ternary match.

Reply:
Removed exact match stage definition and instead added a post ingress stage which provides more flexibility